platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -71,6 +71,16 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslationAnchor) {
  To display circles on the map whose radii correspond to real-world distances,
  use many-sided regular polygons and configure their appearance using an
  `MGLFillStyleLayer` object.
+ 
+ #### Related examples
```

Comment:
This header (and many others in this PR) are generated by `make darwin-style-code`, so much of the PR will be wiped away the next time someone runs that command. Make these changes in the .ejs files in platform/darwin/src/, then run `make darwin-style-code` to verify that they get generated as expected.